sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2769,11 +2823,12 @@ public PDone expand(PCollection<TableRow> input) {
       // BigQuery built-in best effort de-dup mechanism.
 
       // To use this mechanism, each input TableRow is tagged with a generated
-      // unique id, which is then passed to BigQuery and used to ignore duplicates.
+      // unique id, which is then passed to BigQuery and usedTableRowInfo to ignore duplicates.
 
       PCollection<KV<ShardedKey<String>, TableRowInfo>> tagged = input.apply(ParDo.of(
-          new TagWithUniqueIdsAndTable(input.getPipeline().getOptions().as(BigQueryOptions.class),
-              write.getTable(), write.getTableRefFunction())));
+          new TagWithUniqueIdsAndTable<T>(input.getPipeline().getOptions().as(
```

Comment:
Reformat this statement - the line breaking of the first argument looks very strange